Medium/question_6.kt

```diff
@@ -0,0 +1,45 @@
+
```

Comment:
remove this extra space. 
Be consistent across different files.

Reply:
Extra spaces before comments, right?